src/main/java/com/redhat/devtools/intellij/commonUiTestLibrary/fixtures/dialogs/errors/IdeFatalErrorsDialog.java

```diff
@@ -34,20 +29,10 @@ public IdeFatalErrorsDialog(@NotNull RemoteRobot remoteRobot, @NotNull RemoteCom
         super(remoteRobot, remoteComponent);
     }
 
-    public ComponentFixture numberOfExcetionsJBLabel() {
-        List<ComponentFixture> allJBLabels = findAll(ComponentFixture.class, byXpath("//div[@class='JBLabel']"));
-        return allJBLabels.get(0);
-    }
-
-    public ComponentFixture exceptionDescriptionJTextArea() {
-        return find(ComponentFixture.class, byXpath("//div[@class='JTextArea']"));
-    }
-
-    public ComponentFixture previousExceptionButton() {
-        return find(ComponentFixture.class, byXpath("//div[@accessiblename='Previous' and @class='ActionButton'"));
-    }
-
-    public ComponentFixture nextExceptionButton() {
-        return find(ComponentFixture.class, byXpath("//div[@accessiblename='Next' and @class='ActionButton']"));
+    /**
+     * Click on the 'Clear all' button
+     */
+    public void clearAll() {
+        button("Clear all").click();
```

Comment:
I would suggest to use some class for (at least) button labels, so when they change button label, it will be easier to update.

Reply:
Added.